knora/dsplib/utils/knora-schema-resources-only.json

```diff
@@ -0,0 +1,132 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema-resources-only#",
+  "$id": "http://knora.org/pyknora/ontology/knora-schema-resources-only.json",
```

Comment:
what exactly is this? the link isn't pointing anywhere, and as a namespace it doesn't make much sense, as we don't use `pyknora` anywhere.

Reply:
As far as I understood this is used as a unique identifier (https://json-schema.org/understanding-json-schema/basics.html#declaring-a-unique-identifier) and doesn't have to point anywhere. It seems to be good practice to use a URL and I've just copied the schema with pyknora etc. from other schemas used in dsp-tools.

However, I removed the `07` in the schema reference.

Reply:
fair enough, though the URL choice seems weird to me.

Are you sure you can leave out the draft number? also, `schema-resources-only` I have never seen before either. Does validation really work as it should, like this?

Reply:
the latest schema draft seems to be `"$schema": "https://json-schema.org/draft/2020-12/schema"`

Reply:
I see, I thought this was some sort of internal versioning from knora but I see now that this is the official json-schema versioning. So, you are right, it's correct to use the form it had before. I will change it back.

Reply:
(I use draft-07 instead of the newest schema, as I don't know what has changed since - I will create a new yt task to once check and update all the schemas written for dsp-tools)

Reply:
sounds good